frontend/src/components/Banner.tsx

```diff
@@ -151,8 +153,8 @@ export const BannerCta = (props: BannerCtaProps) => {
     );
   }
 
-  // When given a relative URL to link to in `target`, render an <a> (via <Link>):
-  if (props.target.startsWith("/")) {
+  // When given a relative URL to link to in `target`, or metrics are disabled, render an <a> (via <Link>):
+  if (props.target.startsWith("/") || !metricsEnabled) {
```

Comment:
question (blocking): Why does this code care about `metricsEnabled` when deciding whether to use an `OutboundLink` vs. a `Link` component?

Reply:
An [OutboundLink](https://github.com/react-ga/react-ga/blob/master/src/components/OutboundLink.js) is a component provided by `react-ga` to send an event when a link is clicked. If metrics are enabled, the code needs to use a standard React `<Link>` component instead of an `OutboundLink`. We already have code to do this, but only used it for internal links before.